workflow/rules/annotate_cnv.smk

```diff
@@ -0,0 +1,38 @@
+# vim: syntax=python tabstop=4 expandtab
+# coding: utf-8
+
+__author__ = "Patrik Smeds, Almlöf"
+__copyright__ = "Copyright 2021, Patrik Smeds, Jonas Almlöf"
+__email__ = "patrik.smeds@scilifelab.uu.se, jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL3"
+
+
+rule annotate_cnv:
+    input:
+        vcf="{file}.vcf",
+    output:
+        vcf=temp("{file}.annotate_cnv.{tag}.vcf.gz"),
+    params:
+        bed=lambda wildcards: config["annotate_cnv"][wildcards.tag],
+    log:
+        "{file}.annotate_cnv.{tag}.log",
```

Comment:
Should reflect the whole name of the output file, such as `{file}.annotate_cnv.{tag}.vcf.gz.log`